parser.y

```diff
@@ -1607,6 +1607,33 @@ AlterTableSpec:
 		yylex.AppendError(yylex.Errorf("The WITH/WITHOUT VALIDATION clause is parsed but ignored by all storage engines."))
 		parser.lastErrorAsWarn()
 	}
+| "ALTER" "CHECK" Identifier EnforcedOrNot
```

Comment:
```suggestion
|	"ALTER" "CHECK" Identifier EnforcedOrNot
```
Sorry for my unclear suggestion. I mean here should be a tab, as most of the other productions.